### PR TITLE
Implement remote strings for file preview

### PR DESCRIPTION
**Pull request by andrews-moc**
[MOB-2564](https://glia.atlassian.net/browse/MOB-2564)

[MOB-2564]: https://glia.atlassian.net/browse/MOB-2564?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: widgetssdk/src/main/res/layout/file_preview_activity.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:background="@color/glia_black_color"
-        app:navigationContentDescription="@string/glia_top_app_bar_navigate_up_content_description"
+        app:navigationContentDescription="@string/android_app_bar_nav_up_accessibility"
```
Comment (Pelkar) [inline widgetssdk/src/main/res/layout/file_preview_activity.xml:17]: All content descriptions should be setup in code, not in xml

File: widgetssdk/src/main/res/values/new_strings.xml [hunk 2]
```diff
@@ -25,10 +25,18 @@
...
+    <string name="android_preview_save_success_message">@string/glia_preview_activity_image_save_success_msg</string>
+    <string name="android_preview_save_error_message">@string/glia_preview_activity_image_save_fail_msg</string>
+    <string name="android_preview_failed_message">@string/glia_preview_activity_preview_failed_msg</string>
+    <string name="android_preview_accessibility_image">@string/glia_preview_activity_image_content_description</string>
+
     <!-- General -->
     <string name="general_refresh">@string/glia_visitor_code_refresh_button</string>
     <string name="general_accessibility_close">Close Button</string>
     <string name="general_powered_by">@string/glia_chat_glia_logo_content_description</string>
+    <string name="android_app_bar_nav_up_accessibility">@string/glia_top_app_bar_navigate_up_content_description</string>
```
Comment (Pelkar) [inline widgetssdk/src/main/res/values/new_strings.xml:39]: Good catch, this is more general and should not be under any specific feature

**Review by Pelkar — CHANGES_REQUESTED** [review]
One small thing needs to be changed, everything else is good.

File: widgetssdk/src/main/res/layout/file_preview_activity.xml (hunk 1 above)
Comment (andrews-moc) [reply widgetssdk/src/main/res/layout/file_preview_activity.xml]: Fixed 09aa49436d2920b5797bdab2d56eb1ad29c3cdd2
Please have a look 😉

**Review by AndriiHorishniiMOC — APPROVED** [review]
LGTM!

**Review by Pelkar — APPROVED** [review]
LGTM

**Pull request merged**